showcase/app/templates/components/time.hbs

```diff
@@ -0,0 +1,106 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Time Component"}}
+
+<Shw::Text::H1>Time</Shw::Text::H1>
+
+{{! Time component does not include styling so we do not test in Percy at this time }}
+
+<section>
```

Comment:
Suggestion: from @dchyun's comment, could be good to have an example of the Time component in a large paragraph just in case.